rollbar/__init__.py

```diff
@@ -1216,8 +1245,11 @@ def _parse_response(path, access_token, params, resp, endpoint=None):
         log.warning("Rollbar: over rate limit, data was dropped. Payload was: %r", params)
         return
     elif resp.status_code == 413:
-        log.warning("Rollbar: request entity too large. Payload was: %r", params)
-        return
+        payload = json.loads(params)
+        uuid = payload['data']['uuid']
+        host = payload['data']['server']['host']
```

Comment:
Probably would be a good idea to try-except around each of lines 1248-1250, and default `uuid` and `host` to `None` if we can't get them. We don't want to have an exception while trying to gather data for the failsafe...